src/app/app.component.html

```diff
@@ -1,3 +1,4 @@
+<app-confirmation-modal *ngIf="isConfirmModal" [message]='confirmMessage' (close)='confirmed($event)'></app-confirmation-modal>
```

Comment:
@Harshwardhan18 Would recommend the boolean to be renamed to something more informative like- `confirmModalOpen`